region/tourian/main/Metroid Room 1.json

```diff
@@ -1645,6 +1658,34 @@
       ],
       "note": "Bomb Jump between the two floating platforms."
     },
+    {
+      "link": [4, 1],
+      "name": "Use Flash Suit",
+      "requires": [
+        "canMetroidAvoid",
+        "canCameraManip",
+        "canDownGrab",
+        "canHorizontalShinespark",
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 21, "excessFrames": 0}}
```

Comment:
```suggestion
        {"shinespark": {"frames": 21, "excessFrames": 0}},
        {"or": [
          "canInsaneJump",
          {"and": [
            "canTrickyJump",
            "canLateralMidAirMorph"
          ]},
          {"and": [
            "canTrickyJump",
            "canPreciseWalljump",
            {"acidFrames": 16}
          ]},
          {"and": [
            "Gravity",
            {"acidFrames": 16}
          ]},
          {"acidFrames": 27}
        ]}
```

Reply:
An overcomplicated setup to use a flash suit as a way to save some energy.  I prefer long jumping over the acid and diagonal sparking at the last moment - then the metroid flys underneath you and you save 12 energy.

You're crouch jump downgrab is the better logical strat since its more consistent.  I don't see how you move the camera enough without moonwalk

Reply:
Jumping from left platform to runway 1 isn't trivial.

Reply:
The low diagonal spark seems much easier to me. I was sparking much too early before.